### PR TITLE
Cancel a migration in progress

### PR DESCRIPTION
**Pull request by apinnick**
https://issues.redhat.com/browse/MIG-156

CP for 4.2, 4.3, 4.4, 4.5

### CODE REVIEW

**Review by sergiordlr — APPROVED** [review]
Looks good to me.

Thank you!

File: modules/migration-running-migration-plan-cam.adoc [hunk 1]
```diff
@@ -31,6 +31,7 @@ You can run *Stage* multiple times to reduce the actual migration time.
 
 . Optional: In the *Migrate* window, you can select *Do not stop applications on the source cluster during migration*.
 . Click *Migrate*.
+. Optional: To stop a migration in progress, click {kebab} and select *Cancel*.
```
Comment (kalexand-rh) [inline modules/migration-running-migration-plan-cam.adoc:34]: s/{kebab}/Options menu {kebab}

Comment (apinnick) [reply modules/migration-running-migration-plan-cam.adoc]: Done

**Review by kalexand-rh — COMMENTED** [review]
LGTM. Thanks!

**Pull request merged**